app/code/extension/SiteConfigExtension.php

```diff
@@ -128,5 +131,15 @@ public function updateCMSFields(FieldList $fields)
         $sponsorLogo2->getValidator()->setAllowedExtensions(['jpg','jpeg','png']);
         $sponsorLogo3->getValidator()->setAllowedExtensions(['jpg','jpeg','png']);
         $sponsorLogo4->getValidator()->setAllowedExtensions(['jpg','jpeg','png']);
+
+        $fields->addFieldToTab('Root.Alert', TextField::create(
+            'AlertTitle',
+            'Alert Title'
+        ));
+
```

Comment:
Maybe add comments to distinguish the various key components e.g. `footer`,`header` etc